index.html

```diff
@@ -56,7 +52,161 @@ <h4>I’am a Software Developer</h4>
                 don’t hestiate to contact me.</p>
         </div>
         <div class="headline-ilustration">
-            <img src="images/Illustration-header.svg" alt="ilustration image">
+            <img src="images/Illustration-header.svg" class="headline-img" alt="ilustration image">
+        </div>
+    </section>
+    <section class="works">
+        <div class="works-title">
+            <h3 class="work-title">My Recent Works</h3>
+            <hr class="work-line-title">
+        </div>
+        <div class="card-works">
+            <div class="initial-section">
+                <div class="principal-image"><img src="images/project1.svg" class="img1" alt="first work"></div>
+                <div class="content">
+                    <h3 class="header-content">Multi-Post Stories</h3>
+                    <p class="content-paragraph">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the
+                        industry's standard dummy text ever since the 1500s, when an unknown printer took a standard
+                        dummy
+                        text.</p>
+                    <div class="tag-container">
+                        <span class="tag">css</span>
+                        <span class="tag">html</span>
+                        <span class="tag">bootstrap</span>
+                        <span class="tag">Ruby</span>
+                    </div>
```

Comment:
- [ ] Please, consider that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`